bundles/managed-api-service/1.27.0/manifests/managed-api-service.clusterserviceversion.yaml

```diff
@@ -57,692 +57,694 @@ metadata:
     support: RHOAM
-    containerImages: |-
-      [
-        {"component_name":"3scale-operator.v0.9.0","component_url":"registry.redhat.io/3scale-amp2/3scale-rhel7-operator@sha256:b6805840aed8a3cdd58b25383a821c1f85bb4c1959259d9ac4812e56d6651202"},{"component_name":"backend","component_url":"registry.redhat.io/3scale-amp2/backend-rhel8@sha256:d08479d6a1607099f9a125bd21c3c8a685520e35c31aa799a57f211b3192389b"},{"component_name":"apicast","component_url":"registry.redhat.io/3scale-amp2/apicast-gateway-rhel8@sha256:d7af57e97f10c4f2d8ac1ae8f4c4436b6323d2d2ba60dbbb3b50d6328c92133f"},{"component_name":"system","component_url":"registry.redhat.io/3scale-amp2/system-rhel7@sha256:27904327f838ab872c6adf1e4a025b8cd0ff1ca425bfc63816c5ec2253cb79b5"},{"component_name":"zync","component_url":"registry.redhat.io/3scale-amp2/zync-rhel8@sha256:c231dfc49feb71f250a30e6f724fc29eefea3cf30b8bf92547fc28e616f01864"},{"component_name":"system_memcached","component_url":"registry.redhat.io/3scale-amp2/memcached-rhel7@sha256:4a69109a33167ac9a13088d1da608e32feb31cc394be1c350a7d36e1d88882d3"},{"component_name":"cloud-resources.v0.40.0","component_url":"quay.io/integreatly/cloud-resource-operator:v0.40.0"},{"component_name":"grafana-operator.v4.2.0","component_url":"quay.io/grafana-operator/grafana-operator:v4.2.0"},{"component_name":"marin3r.v0.10.0","component_url":"quay.io/3scale/marin3r:v0.10.0"},{"component_name":"rhsso-operator.7.6.0-opr-003","component_url":"registry.redhat.io/rh-sso-7/sso7-rhel8-operator@sha256:f69187d16e22f42173b99a1c4f0ce6cd212260da263b4a7a1eb08e5822cf4f66"},{"component_name":"rhsso_openjdk","component_url":"registry.redhat.io/rh-sso-7/sso76-openshift-rhel8@sha256:6a2507d7f42f7eb9f71087179f1c6464dd45a4c0cb9d82999b10c2e008427f37"},{"component_name":"keycloak_init_container","component_url":"registry.redhat.io/rh-sso-7/sso7-rhel8-init-container@sha256:7cfd440a9040f1fdeb0f3c30b7e283cd907671b57e8bec19a8c3249a76904c43"},{"component_name":"observability-operator.v3.0.13","component_url":"quay.io/rhoas/observability-operator:v3.0.13"},{"component_name":"3scale-openshift-servce-mesh","component_url":"registry.redhat.io/openshift-service-mesh/proxyv2-rhel8:2.1.2-2"},{"component_name":"marin3r-limitador","component_url":"quay.io/3scale/limitador:v0.5.1"},{"component_name":"observability-grafana-plugins-init","component_url":"quay.io/grafana-operator/grafana_plugins_init:0.0.6"},{"component_name":"observability-origin-oauth-proxy","component_url":"registry.redhat.io/openshift4/ose-oauth-proxy@sha256:9a5ee95f8e99a63a4ad0e8b01683ac03c75337bbbe3d504d199a97f9921eb0c1"},{"component_name":"observability-prometheus-operator","component_url":"quay.io/prometheus-operator/prometheus-operator@sha256:066fce4a6b7392f07f7179b59ed4448bacc0767277637de99809449637be924b"},{"component_name":"rhsso-operator","component_url":"registry.redhat.io/rh-sso-7/sso7-rhel8-operator@sha256:f69187d16e22f42173b99a1c4f0ce6cd212260da263b4a7a1eb08e5822cf4f66"}]
-    serviceAffecting: "true"
```

Comment:
Would this prevent the CVE pipeline from working correctly ? :thinking:

Reply:
possibly can add it back in, its all generated from the make bundle command but I have had to make a number of changes after it was run

Reply:
Oh okay, I think it's the `make release` command that adds the container images annotation :thinking: 
```
 OLM_TYPE=managed-api-service SEMVER=1.27.0 IMAGE_TAG=master SERVICE_AFFECTING=true make release/prepare 
```

Reply:
added them back in manually, not sure why github is not showing this as outdated as its gone as a change in the files view.